pkg/testhelper/testhelper.go

```diff
@@ -71,3 +71,14 @@ func SkipIfEmptyAll(skip func(string, ...int), object ...interface{}) {
 		skip(fmt.Sprintf("Test skipped because there are no %s to test, please check under test labels", allTypes))
 	}
 }
+
+func AddContainerResultLog(prefix string, object interface{}, log func(string, ...interface{}), fail func(string, ...int)) {
```

Comment:
Great idea to avoid repeating the same code all over the test suites. But I'd use a more generic name because it can also be used for Pods, Namespaces or any other resource. Something like `AddTestResultLog()` or similar.